test/es2015-module-options.test.js

```diff
@@ -0,0 +1,155 @@
+import UglifyJsPlugin from '../src/index';
+import {
+  cleanErrorStack,
+  createCompiler,
+  compile,
+} from './helpers';
+
+describe('when applied with uglifyOptions.ecma', () => {
+  it('matches snapshot for ecma 5', () => {
+    const compiler = createCompiler({
+      entry: `${__dirname}/stubs/es-entry.js`,
+      output: {
+        path: `${__dirname}/dist-es`,
+        filename: '[name].[chunkhash].js',
```

Comment:
[chunkhash] help with unique file names for the snapshotted files tests l, reads better when they change